frontend/plugins/pos_ui/src/modules/create-pos/components/sync/sync.tsx

```
@@ -110,6 +137,7 @@ export default function SyncCardForm({
             onClick={handleToggleNewConfig}
             className="bg-indigo-600 hover:bg-indigo-700 text-white flex items-center gap-2"
             disabled={isReadOnly}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)